src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1020,6 +1021,33 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode, callable $e
                             ->end()
                             ->defaultValue([])
                         ->end()
+                        ->arrayNode('globals')
+                            ->info('Global parameters.')
+                            ->example(['app_version' => 3.14])
+                            ->normalizeKeys(false)
+                            ->useAttributeAsKey('name')
+                            ->arrayPrototype()
+                                ->fixXmlConfig('parameter')
+                                ->children()
+                                    ->variableNode('value')->end()
+                                    ->stringNode('message')->end()
+                                    ->arrayNode('parameters')
+                                        ->normalizeKeys(false)
+                                        ->useAttributeAsKey('name')
+                                        ->scalarPrototype()->end()
+                                    ->end()
+                                    ->stringNode('domain')->end()
+                                ->end()
+                                ->beforeNormalization()
+                                    ->ifTrue(fn ($v) => !\is_array($v))
+                                    ->then(fn ($v): array => ['value' => $v])
```

Comment:
the closures could all be made static also
```suggestion
                                    ->then(fn ($v) => ['value' => $v])
```